api/api_integration_test.go

```diff
@@ -87,6 +88,10 @@ func Test_Integration(t *testing.T) {
 		cryptoInstance.Config.Fspath = "../testdata/tmp"
 		cryptoInstance.Configure()
 
+		networkInstance := pkg2.NetworkInstance()
```

Comment:
So the registry uses the network instance but doesn't configure it....
Spaghetti in the making

Reply:
Well it does use it but through the singleton construction, so it must be configured before registry is configured :-(